src/main/steps/common/progress-bar.njk

```diff
@@ -35,14 +35,14 @@
     label: {
       html: textWithVisuallyHiddenSpan(submitted, displayState.isAtOrAfter('Submitted'))
     },
-    complete: displayState.isAtOrAfter('Submitted'),
-    active: displayState.at('AwaitingDocuments')
+    complete: displayState.isAtOrAfter('Submitted') or displayState.at('AwaitingHWFDecision'),
+    active: displayState.at('AwaitingDocuments') and not (displayState.isAtOrAfter('Submitted') or displayState.at('AwaitingHWFDecision'))
   },  {
     label: {
       html: textWithVisuallyHiddenSpan(courtChecks, displayState.isBefore('Submitted'))
     },
     complete: displayState.isAfter('Submitted'),
-    active: displayState.isAtOrAfter('AwaitingAos')
+    active: displayState.isAfter('AwaitingAos') or displayState.at('AwaitingHWFDecision')
```

Comment:
Should we still be using `isAtOrAfter('AwaitingAos')` rather than `isAfter('AwaitingAos')`?

Reply:
Yes that's true we need isAtOrAfter 👍